src/commonComponents/iconWithLabel/IconWithLabel.js

```diff
@@ -0,0 +1,34 @@
+// Library
+import PropTypes from 'prop-types'
+
+// css
+import './iconWithLabel.css'
+
+function FavouriteFilter(props){
+
+    const { icon, label, containerClassName, imgClassName } = props;
+
+    return(
+        <div className={containerClassName}>
+            <img src={icon} alt={label} className={imgClassName} />
```

Comment:
your defaultProps for icon is null, it won't be visible on the screen that's fine because it will have 0 * 0 dimension but it's still be there in the DOM you can check, so better keep a conditional check when you want to show it